### ISSUE TITLE
[mysqldef] Comments on table not reflected

### ISSUE THREAD

**Issue by masaki4680** [k0kubun/sqldef#271]
Comments on table not reflected

Table comments were not reflected

## Input SQL

```sql
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='都道府県マスター';
```

## Reflected SQL

```sql
-- carTrade.prefecture definition

CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

## Expected output

I expect to reflect table comments

**Comment by k0kubun** [comment]
I can't reproduce it. Please follow the issue template exactly. For example,

## --export output

```
-- No table exists --
```

## Input SQL
```
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='都道府県マスター';
```

## Current output
```
-- Apply --
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='都道府県マスター';
```

## Expected output
```
-- Apply --
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='都道府県マスター';
```

---

So it actually works for me.

**Comment by masaki4680** [comment]
sorry!

## --export output
```sql
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

## Input SQL

```sql
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='都道府県マスター';
```
## Current output
```bash
-- Nothing is modified --
```
## Expected output
```sql
CREATE TABLE `prefecture` (
  `id` int(11) unsigned NOT NULL COMMENT 'ID',
  `name` varchar(50) NOT NULL COMMENT '都道府県名',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='都道府県マスター';
```

**Comment by k0kubun** [comment]
Thanks 🙂 Now I can understand what it is. 

So this issue is effectively a feature request to support `ALTER TABLE xxx COMMENT = 'yyy';`. I'll find time to work on this.

**Comment by masaki4680** [comment]
I understand that.please take your time.

**Comment by k0kubun** [comment]
Fixed in v0.13.6.
